common/junit-platform-native/src/main/java/org/graalvm/junit/platform/JUnitPlatformFeature.java

```diff
@@ -74,66 +75,73 @@
 public final class JUnitPlatformFeature implements Feature {
```

Comment:
If we migrate this class and it's related classes (e.g. `NativeImageJUnitLauncher`, ...) to e.g. `com.oracle.svm.junit5` in e.g.
`graal/substratevm/src/com.oracle.svm.junit5/src/com/oracle/svm/junit5` in
https://github.com/oracle/graal
we would be able to support building junit5 tests into images even without `native-build-tools`.

This would be very handy to eventually replace `mx native-unittest` with `mx native-unittest5` in the future and to only have one remaining junit feature that is used for all situations.

cc @zapster 

(obviously this would happen via follow up PRs)